docs/DateTimePicker.md

```diff
@@ -0,0 +1,31 @@
+# DateTimePicker
+
+> React Native date & time picker component for iOS, Android and Windows
+
+## Props
+
+### `value (required)`
+
+Type: `Date`
+
+Defines the date or time value used in the component.
+
+### `onChange`
+
+Type: `(value: Date) => void`
+
+Date change handler.
+
+This is called when the user changes the date or time in the UI. It receives the date as parameters.
+
+### `onBlur`
+
+Type: `(event: NativeSyntheticEvent<TextInputEndEditingEventData>) => void`
```

Comment:
This generic value shouldn't be `TextInputEndEditingEventData`, please hover up on the onBlur props then we can know certainly what is the generic value.